addons/pos_epson_printer/static/src/js/printers.js

```diff
@@ -22,9 +45,10 @@ var EpsonPrinter = core.Class.extend(PrinterMixin, {
         if ((resultConnect == 'OK') || (resultConnect == 'SSL_CONNECT_OK')) {
             this.ePOSDevice.createDevice(deviceId, this.ePOSDevice.DEVICE_TYPE_PRINTER, options, this.callback_createDevice.bind(this));
         } else {
-            Gui.showPopup('ErrorPopup', {
-                'title': _t('Connection to the printer failed'),
-                'body':  _t('Please check if the printer is still connected, if the configured IP address is correct and if your printer supports the ePOS protocol.'),
+            Gui.showPopup('EpsonPrinterErrorPopup', {
+                title: _t('Connection to printer failed'),
+                url: this.ePOSDevice.conectionObj.address_p,
```

Comment:
This template to add a link to documentation is maybe overkill
I think that only a message more explicit is the best solution.

Reply:
@qle-odoo code updated :)